src/data/resolvers/colonyActions.ts

```diff
@@ -16,6 +16,7 @@ interface EventValue {
   token: Address;
   fromPot: BigNumberish;
   toPot: BigNumberish;
+  who: Address;
```

Comment:
I know this comes from the event, but I so **hate** this prop name!

Reply:
Same here, it's not a great name definitely hahaha